build_psa_compliance.py

```diff
@@ -40,6 +40,11 @@
         "tgt_dev_apis_tfm_musca_b1",
         "tgt_ff_tfm_musca_b1",
     ],
+    "ARM_MUSCA_S1": [
```

Comment:
Is this target, able to run IPC or Firmware Framework test suite?  If not please handle it and raise an issue that could be addressed in the near future.

Reply:
I don't see - `tgt_ff_tfm_musca_s1` in  https://github.com/ARM-software/psa-arch-tests/tree/master/api-tests/platform/targets

Reply:
> 
> 
> I don't see - `tgt_ff_tfm_musca_s1` in https://github.com/ARM-software/psa-arch-tests/tree/master/api-tests/platform/targets

Yes, you are right. I removed it.

Reply:
When referring - https://github.com/ARMmbed/mbed-os-tf-m-regression-tests/blob/master/build_psa_compliance.py#L310, As ARM_MUSCA_S1 doesn't support FF/IPC, it needs to be handled and the new issue raised.

Reply:
Issue [#49](https://github.com/ARMmbed/mbed-os-tf-m-regression-tests/issues/49) is raised, IPC is handled below.